components/OfflineIndicator.js

```diff
@@ -0,0 +1,50 @@
+const T = require('prop-types');
+const { default: Styled } = require('styled-components');
+const { useSafeAreaInsets } = require('react-native-safe-area-context');
+const { default: Icons } = require('@expo/vector-icons/Feather');
+const { Text } = require('components/Type');
+const Theme = require('theme');
+
+const internals = {};
+
+module.exports = function OfflineIndicator(props) {
+
+    const {
+        BannerContainer,
+        BannerText
+    } = internals;
+
+    const insets = useSafeAreaInsets();
+
+    return (
+        <BannerContainer $insets={insets.top}>
+            <Icons name='alert-triangle' style={{ marginRight: Theme.spacing(1) }}></Icons><BannerText>Check network connection</BannerText>
+        </BannerContainer>
+    );
+};
+
+internals.BannerContainer = Styled.View`
+    position: absolute;
+    min-width: 250;
```

Comment:
```suggestion
    min-width: 250px;
```

only because it's giving me warnings 😅